pkg/infoschema/tables.go

```diff
@@ -1663,6 +1666,22 @@ var tableKeywords = []columnInfo{
 	{name: "RESERVED", tp: mysql.TypeLong, size: 11},
 }
 
+var tableTiDBParamsCols = []columnInfo{
+	{name: "COMPONENT", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
+	{name: "TYPE", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
+	{name: "NAME", tp: mysql.TypeVarchar, size: 256, flag: mysql.NotNullFlag},
```

Comment:
`VARIABLES_INFO` has `VARIABLE_NAME` instead of `NAME`, maybe we should follow this naming scheme. 

```suggestion
	{name: "PARAM_NAME", tp: mysql.TypeVarchar, size: 256, flag: mysql.NotNullFlag},
```